### PR TITLE
Fix bad initialization of TableRowSorters.

### PR DESCRIPTION
**Pull request by rhwood**
@petecressman These are the changes needed to fix #1980 without resorting to the use of ```jmri.util.com.sun.TableSorter```.

### CODE REVIEW

File: java/src/jmri/jmrit/beantable/oblock/TableFrames.java [hunk 1]
```diff
@@ -528,9 +528,9 @@ protected JInternalFrame makePortalFrame() {
 //        _portalModel.init();
         _portalTable = new DnDJTable(_portalModel, new int[]{PortalTableModel.DELETE_COL});
 //        _portalModel.makeSorter(portalTable);
-        TableRowSorter<OBlockTableModel> sorter = new TableRowSorter<>();
-        sorter.setComparator(OBlockTableModel.SYSNAMECOL, new jmri.util.SystemNameComparator());
-        _oBlockTable.setRowSorter(sorter);
+        TableRowSorter<PortalTableModel> sorter = new TableRowSorter<>(_portalModel);
+        sorter.setComparator(PortalTableModel.SYSNAMECOL, new jmri.util.SystemNameComparator());
```
Comment (rhwood) [inline java/src/jmri/jmrit/beantable/oblock/TableFrames.java]: Why do you *not* want the SystemName column to be sortable using the SystemNameComparator (which is designed to allow SystemNames to be sortable)?

Comment (petecressman) [inline java/src/jmri/jmrit/beantable/oblock/TableFrames.java]: There is no SYSNAMECOL in PortalTableModel or SignalTableModel.
The problem lies in lines 533 and 603.
How can i correct this?

Comment (rhwood) [inline java/src/jmri/jmrit/beantable/oblock/TableFrames.java]: SYSCOLNAME is a field defined in PortalTableModel or its super class, but is not defined in SignalTableModel (which is why I do not have a comparator set for it when setting up the _signalTable).

Lines 533 and 603 associate a sorter with a table, so unless these tables should not be sortable at all, I don't know what you mean by a problem in those lines (but if they weren't to be sorted, they would not have needed the old TableSorter models either).

**Comment by rhwood** [comment]
Closing as OBE (accidentally picked up all changes in JMRI/master).

**Pull request closed without merging**